modules/cluster/variables.tf

```diff
@@ -354,6 +354,12 @@ variable "create_autoscaler_role" {
   default     = true
 }
 
+variable "create_secrets_role" {
+  description = "Flag to control cluster secrets iam role creation"
+  type        = bool
+  default     = true
```

Comment:
Let's not set this to true, unless we want to make a breaking change and not create any vault resources. If I just follow the simple example outlined in the read me, it will create a vault instance and secrets manager iam role.